redaxo/src/core/fragments/core/dropdowns/dropdown.php

```diff
@@ -4,32 +4,45 @@
  * @psalm-scope-this rex_fragment
  */
 
-$toolbar = isset($this->toolbar) && $this->toolbar;
-$group = isset($this->group) && $this->group;
+$toolbar = isset($this->toolbar) && $this->toolbar ? true : false;
+$group = isset($this->group) && $this->group ? true : false;
+$module_dropdown_id = 'module-dropdown-'.rand(999,999999999);
```

Comment:
Bist du hier nicht im falschen Fragment? Der Core kennt keine Module-Id.

Reply:
Evtl. ist die Variable `$module_dropdown_id` etwas irreführend benannt. Es wird ja auch einfach nur random eine ID verteilt und nicht z.B. auf die REX-Slide-ID zurückgegriffen. Das JS für die Suche benötigt einfach nur eine eindeutige ID:
`var dropdownId = $(this).closest('ul').attr('id');`
https://github.com/redaxo/redaxo/pull/4722/files/3a72cf6cec7e48e659d0e2a927013a99bc82d9df#diff-a6ec236c7df315a2f2f122d36e81950949f552e93004b85549854674efce1446R46

Reply:
Das bedeutet, dass die Suche bei jedem Dropdown erscheint? Ab wieviel Items wird diese aktuell ausgegeben?

Reply:
Ja, aktuell bekommt da jedes Dropdown eine Suche - was nicht so sinnvoll ist. 
Wir hatten die Diskussion schon weiter oben, aber haben keine konkrete Zahl festgelegt.
Mein Vorschlag wäre ab 5:

https://github.com/redaxo/redaxo/pull/4722#issuecomment-880806379